src/GongSolutions.WPF.DragDrop/DefaultDropHandler.cs

```diff
@@ -234,6 +252,30 @@ public virtual void Drop(IDropInfo dropInfo)
             }
         }
 
+        private static void Move(IList list, int sourceIndex, int destinationIndex)
+        {
+            if (!IsObservableCollection(list))
+                throw new ArgumentException("ObservableCollection<T> was expected",nameof(list));
```

Comment:
Should have a space between parameters.
```suggestion
                throw new ArgumentException("ObservableCollection<T> was expected", nameof(list));
```